### ISSUE TITLE
Cannot add Locations of differing worlds

### ISSUE THREAD

**Issue by pkristian** [JamesPeters98/ChestsPlusPlus#13]
Hello, i updated your wonderfull plugin from 2.1 to 2.2 and came to existence following issue:

server version: paper-1.16.1-123

it dumps in log again and again

```
[12:52:11] [Server thread/WARN]: [ChestsPlusPlus] Task #506 for ChestsPlusPlus v2.2-Release generated an exception
java.lang.IllegalArgumentException: Cannot add Locations of differing worlds
	at org.bukkit.Location.subtract(Location.java:412) ~[patched_1.16.1.jar:git-Paper-123]
	at com.jamesdpeters.minecraft.chests.misc.Utils.isLocationInViewDistance(Utils.java:271) ~[?:?]
	at com.jamesdpeters.minecraft.chests.storage.abstracts.AbstractStorage.lambda$updateSign$2(AbstractStorage.java:143) ~[?:?]
	at java.util.ArrayList.forEach(ArrayList.java:1257) ~[?:1.8.0_231]
	at com.jamesdpeters.minecraft.chests.storage.abstracts.AbstractStorage.lambda$updateSign$3(AbstractStorage.java:142) ~[?:?]
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:1.8.0_231]
	at java.util.Collections$UnmodifiableCollection.forEach(Collections.java:1082) ~[?:1.8.0_231]
	at com.jamesdpeters.minecraft.chests.storage.abstracts.AbstractStorage.updateSign(AbstractStorage.java:140) ~[?:?]
	at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftTask.run(CraftTask.java:99) ~[patched_1.16.1.jar:git-Paper-123]
	at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468) ~[patched_1.16.1.jar:git-Paper-123]
	at net.minecraft.server.v1_16_R1.MinecraftServer.b(MinecraftServer.java:1291) ~[patched_1.16.1.jar:git-Paper-123]
	at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:377) ~[patched_1.16.1.jar:git-Paper-123]
	at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1212) ~[patched_1.16.1.jar:git-Paper-123]
	at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1000) ~[patched_1.16.1.jar:git-Paper-123]
	at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Paper-123]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_231]
```

**Comment by JamesPeters98** [comment]
Yep I found this yesterday! I'll be releasing a new update to fix it 👍

**Comment by pkristian** [comment]
Great, looking for this. Keep up with this great work. Thanks :)

**Comment by JamesPeters98** [comment]
This is now fixed in version 2.2.1 on Bukkit.
